data/locations.json

```diff
@@ -0,0 +1,108 @@
+{
+  "locations": [
+    {
+      "id": 1,
+      "uuid": "0e5c7eba-13fd-42b8-a4ae-5d973958a160"
```

Comment:
Why do we need hard-coded uuids?

Reply:
I took part of the real data that we filled in, and this link with the locations, I need it to fill in the city and address models